### ISSUE TITLE
Session ends right after login

### ISSUE THREAD

**Issue by vranki** [Awesome-Technologies/synapse-admin#135]
After a successful login, synapse admin returns to the login screen with message "Your session has ended, please reconnect.".

(Tested with latest docker image)

**Comment by vranki** [comment]
I was not admin in the database when this happened. After setting admin bit it no longer does this. Not sure if the tool is intended to be usable without admin bit but at least the error message is wrong.

**Comment by awesome-michael** [comment]
No the tool is just for server admins. We can try to adjust the error message in this case.

**Comment by vranki** [comment]
Yes, an acceptable fix would be for example error "You are not and administrator in this server. Please add yourself as administrator by following these instructions <good link to explain how to admin a user>".

**Comment by dklimpel** [comment]
It is a missing upstream feature: https://github.com/marmelab/react-admin/issues/6547#issuecomment-914989962

**Comment by alexander-potemkin** [comment]
Indeed, it would be nice to have a better error message.
